src/jvm/main/org/jetbrains/kotlinx/lincheck/LinChecker.kt

```diff
@@ -185,8 +185,8 @@ class LinChecker (private val testClass: Class<*>, options: Options<*, *>?) {
         parallelExecution.map { it.size }.sum() == 0
 
 
-    private fun CTestConfiguration.createVerifier(checkStateEquivalence: Boolean) =
-        verifierClass.getConstructor(Class::class.java).newInstance(sequentialSpecification).also {
+    private fun CTestConfiguration.createVerifier(checkStateEquivalence: Boolean): Verifier {
+        return verifierClass.getConstructor(Class::class.java).newInstance(sequentialSpecification).also {
```

Comment:
It would be more idiomatic to rollback it to `=` instead of `{ return ... }`

Reply:
Done